front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -165,10 +172,30 @@
         align-items: center;
         gap: 0.5rem;
         margin-bottom: 0.5rem;
+        margin-right: 0.5rem;
         height: 2rem;
         .multi-select-buttons {
           display: flex;
-          gap: 0.5rem;
+        }
+      }
+      .filter-toggle {
+        background-color: transparent;
+        border: none;
+        color: var(--secondary);
+        transition: 0.2s;
+        padding: 0.25rem 0.5rem;
+        font-size: 1rem;
+        border-radius: 4px;
+        &:hover {
+          background-color: var(--secondary);
+          color: var(--white);
+          .scenario-details-modify-button-text {
+            opacity: 1;
+          }
+        }
+        &.on {
+          color: white;
+          background-color: var(--secondary);
```

Comment:
Don't Repeat Yourself! You've already wrote this above, please try to use common classes.